person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..500)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @check_name = Corrector.new
+    @rental = []
   end
```

Comment:
- [ ] Let's create a method for adding rental. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Check out this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) as reference.

Reply:
Oh, it is true! I missed that method, thanks!